src/server/api/schema.js

```diff
@@ -61,6 +60,8 @@ import { change } from "../local-auth-helpers";
 import { symmetricEncrypt } from "./lib/crypto";
 import Twilio from "twilio";
 
+const UpdateQuestionResponses = require("./mutations/updateQuestionResponses");
```

Comment:
please move this into the more standard import inside the "./mutations" two lines down, and just importing the function rather than the file as a whole.

Reply:
This makes it testable.  We can discuss.